lib/elixir/lib/kernel.ex

```diff
@@ -4573,7 +4576,7 @@ defmodule Kernel do
   end
 
   defp split_words(string, [mod])
-  when mod == ?s or mod == ?a or mod == ?c do
+       when mod === ?s or mod === ?a or mod === ?c do
```

Comment:
Let's revert this one. The style inconsistency with the codebase is not worth the very small gains.